.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+## Summary of Issue:
```

Comment:
Seems like this only applies to issues that are bugs.  We have a range of ways that we utilize issues and this template would not be appropriate for all of them.  So, maybe that can be clarified somehow, that this is how users should report a bug?

Reply:
I agree with @dlawrenncar . But I'd go further than this: I have found issue templates (such as the one in place for manage_externals) to be more annoying than helpful, both when submitting an issue and when reading other people's issues. The problem is that many of these sections end up not applying to any given issue, so it just adds extra work for both writing an issue and reading an issue (because you have to read through the sections filled out with N/A or whatever).

If we do anything here, I feel this should not be a template like this (where people feel they need to fill something in for every section), but rather a short paragraph saying something like, "A good bug report contains a brief summary of the issue, followed by more detailed information on how to reproduce the problem... [etc.]"

Reply:
@dlawrenncar good point on the different types. @billsacks I do find templates to be useful. Templates help me to remember important things to include, or prompt users on those things. As long as the user knows it's OK to delete prompts as appropriate I think that's OK. So, I'm working on a compromise. It allows you to specify the general type of issue, and for bugs has a few things to fill out, with directions to remove as appropriate. Since, standard users don't have access to adding labels, having the list of general types will be useful for us to categorize after the issue is created.